### DISCUSSION TITLE
Animate bar charts with wave animation

### DISCUSSION THREAD

**Discussion by ollybondareva** [airbnb/visx#1775] [Q&A]
I have bar chart from visx and I need to apply some kind of wave diagonal animation for it. The animation should go from top left corner to bottom right. Any thoughts how to archive this while using visx bar chart? 

Example of animation I would like to achieve:
<img width="361" alt="Screenshot 2023-12-12 at 11 01 33" src="https://github.com/airbnb/visx/assets/15728593/118191bf-f549-4956-b759-c54ea2b3e8e2">
https://github.com/airbnb/visx/assets/15728593/281e341b-fbc7-4bae-9884-1cc56acd4525

**Comment by williaster** [comment]
you can probably accomplish this with `svg` gradients. `@visx/gradients` doesn't support animation of values, but if you add in some `<animate />` elements to a custom `<linearGradient />` definitions it should be possible.

[sandbox](https://codesandbox.io/p/sandbox/delicate-thunder-rqtr9l?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clqbgc8o500063b6k5gr6x6z2%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clqbgc8o500023b6kalbtw3xm%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clqbgc8o500033b6k6tykypd7%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clqbgc8o500053b6k03oae72h%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B46.7024409850934%252C53.2975590149066%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clqbgc8o500023b6kalbtw3xm%2522%253A%257B%2522id%2522%253A%2522clqbgc8o500023b6kalbtw3xm%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522clqbgc8o500053b6k03oae72h%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqbgc8o500043b6kfqkj4fpc%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clqbgc8o500053b6k03oae72h%2522%252C%2522activeTabId%2522%253A%2522clqbgc8o500043b6kfqkj4fpc%2522%257D%252C%2522clqbgc8o500033b6k6tykypd7%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clqbgc8o500033b6k6tykypd7%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

<img width="1546" alt="image" src="https://github.com/airbnb/visx/assets/4496521/10a0488f-8771-48b7-96c5-63b6e2b5f7a9">
